apps/menu-bar/macos/ExpoMenuBar-macOS/AppDelegate.h

```diff
@@ -4,7 +4,7 @@
 
 @class RCTBridge;
 
-@interface AppDelegate : NSObject <NSApplicationDelegate>
+@interface AppDelegate : NSObject <NSApplicationDelegate, NSUserNotificationCenterDelegate>
```

Comment:
This is actually useful for me to understand how people use the upgrade-helper for macOS, thanks!